boot/kernel_os.go

```diff
@@ -174,3 +176,69 @@ func InUse(name string, rev snap.Revision) bool {
 
 	return false
 }
+
+var (
+	ErrBootNameAndRevisionAgain = errors.New("boot revision not yet established")
+	ErrUnsupportedSnapType      = errors.New("cannot find boot revision for anything but core (or base) and kernel")
```

Comment:
"base (or core)",  also this seems an "internal error",  does it need to be predefined vs Errorf ?